cilium/cmd/bpf_ct_list.go

```diff
@@ -30,11 +30,11 @@ var bpfCtListCmd = &cobra.Command{
 	PreRun: requireEndpointIDorGlobal,
 	Run: func(cmd *cobra.Command, args []string) {
 		if args[0] == "global" {
-			dumpCtProto(ctmap.MapName6+args[0], ctmap.CtTypeIPv6Global)
-			dumpCtProto(ctmap.MapName4+args[0], ctmap.CtTypeIPv4Global)
+			dumpCtProto(ctmap.MapName6Global, "")
```

Comment:
cramp.MapName6Global is assigned the following value: MapName6Global = MapName6 + "global" so there is no need to use args[0] here.